pkg/config/config_template.yaml

```diff
@@ -273,8 +273,14 @@ api_key:
 # dogstatsd_port: 8125
 #
 # The host to bind to receive external metrics (used only by the dogstatsd
-# server for now). For dogstatsd this is ignored if
-# 'dogstatsd_non_local_traffic' is set to true
+# server for now). For dogstatsd this is ignored if 'dogstatsd_non_local_traffic'
+# is set to true. For APM it is ignored if 'apm_config.non_local_traffic' is set
+# to true.
+# In case of the trace-agent, this value is also used as the host to send metrics to.
+# When it's not set, it defaults to "localhost". This can be problematic in environments
+# where Dogstatsd is listening on IPv6 address "::1". It will cause the trace-agent
+# metrics to go missing. In these cases, it is necessary to explicitly set this value
+# to "localhost".
```

Comment:
are you sure that setting to `localhost` will change anything? Because internally the Agent already uses `localhost` as the default value for this setting: https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L122

I'd expect users to have to set the config value to `127.0.0.1` here for example, if they want to avoid binding to `::1`, but I may be missing something

Reply:
Not sure at all 😄 I likely misunderstood the solution that was mentioned on Slack from our support team. Changed this to 127.0.0.1, although that's a bit odd, given that it's now an IP, not a host (option is called "bind_host")

Reply:
(you updated `datadog.example.yaml` with `127.0.0.1` but here `localhost` is still mentioned ;) )

It's a bit unclear to me as well 😄 I've followed up with support to get more details, let's wait for their response

Reply:
Ok, I'll try and reach out.